app/helpers/hyku_knapsack/application_helper.rb

```diff
@@ -13,6 +13,21 @@ def video_embed_viewer_display(work_presenter, locals = {})
     def video_embed_viewer_display_partial(work_presenter)
       'hyrax/base/' + work_presenter.video_embed_viewer.to_s
     end
+
```

Comment:
Thanks for checking this out. This doesn't quite work yet because we need to ask if there are any highlight matches. If so, query should be parent_query. 

ref: https://github.com/scientist-softserv/iiif_print/blob/main/app/views/catalog/_index_header_list_default.html.erb#L6

I'll see if I can get the logic to match.

Reply:
actually, when I click the work from the catalog results I'm not hitting this method 🤔

Reply:
Since iiif_print is a part of hyku can we just update their version of the partial?

Reply:
My preference is to not update the partial from Hyku, if we can rely on a method.  This method would then be back ported.

However, the primary problem is the SolrDocument does not respond to `any_highlighting?`

<img width="1218" alt="Screenshot 2023-10-06 at 9 28 27 AM" src="https://github.com/scientist-softserv/adventist_knapsack/assets/2130/d0b10938-0621-4a77-b1fc-98a7aee49b56">

Reply:
Yes we can.  But it will be in the form of the method I just pushed up.

For the moment, let's see if what I have works.  If it does, then we back port the method.